common/types.d.ts

```diff
@@ -194,6 +201,12 @@ declare global {
   type ILookup = { [key: string]: any }
 
   type IPreferences = ILookup
+
+  type IManufacturer = {
+    name?: string
+    product: { version?: string; name?: string; code?: number }
```

Comment:
Prob should expect product to be optional:

```
  product?: { version?: string; name?: string; code?: number }
```

Same for platform

Reply:
updated